zamba/models/cnnensemble_model.py

```diff
@@ -106,9 +114,11 @@ def predict(self, file_names):
             columns=config.CLASSES,
         )
 
-        blanks = pd.DataFrame(0, index=blank_file_names, columns=preds.columns)
-        blanks["blank"] = 1
-        preds = pd.concat([preds, blanks], axis=0)
+        if "blank" in preds:
+            del preds["blank"]
+
+        preds = preds.join(blank_probabilities, how="outer")
```

Comment:
With `to_frame`, I think we can do `preds = blank_probabilities.join(preds.drop('blank', axis=1)`